tests/JavascriptTests.cmake

```diff
@@ -27,25 +27,10 @@ function(add_eslint_test name input)
     message(FATAL_ERROR "CMake variable ESLINT_EXECUTABLE is not set. Run 'girder-install web --dev' or disable JAVASCRIPT_STYLE_TESTS.")
   endif()
 
-  set(_args ESLINT_IGNORE_FILE ESLINT_CONFIG_FILE)
-  cmake_parse_arguments(fn "${_options}" "${_args}" "${_multival_args}" ${ARGN})
-
-  if(fn_ESLINT_IGNORE_FILE)
-    set(ignore_file "${fn_ESLINT_IGNORE_FILE}")
-  else()
-    set(ignore_file "${PROJECT_SOURCE_DIR}/.eslintignore")
-  endif()
```

Comment:
Unfortunately, I think you'll have to add the ignore file configuration option back.  `eslint` only looks for an ignore file in `CWD`, and there is no way to extend it by adding ignore files in subdirectories like you can with config files.

Reply:
Yes, you're right, I misread the docs. I'll fix.